src/components/NavigationButtons.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Button, SecondaryButton, screenBreakpoints } from './Common'
+import { I } from './Typography'
+
+const StyledButton = styled(Button)`
+  margin: 0;
+
+  @media (max-width: ${screenBreakpoints.xs}px) {
+    margin-top: 0.5em;
```

Comment:
If we remove the `StyledSecondaryButton` then you can change this to `margin-bottom` so it won't add additional space on top :) 
```suggestion
    margin-bottom: 0.5em;
```

Reply:
08d4c9e